include/circt/Dialect/Verif/VerifOps.td

```diff
@@ -40,4 +42,32 @@ def CoverOp : AssertLikeOp<"cover"> {
   let summary = "Ensure that a property can hold.";
 }
 
+def FormatStringOp : VerifOp<"fstr", [
```

Comment:
Maybe calling it `format_string` could be easier to read in the assembly, since we don't really need to have short mnemonics.